From 12e3be1dafccaaba1a8b6dde8a6adb36a662adc0 Mon Sep 17 00:00:00 2001
From: m <m@l>
Date: Sat, 15 Aug 2026 19:57:06 -0400
Subject: [PATCH] 
 5-models-server/0001-WIP-add-TurboQuant-KV-cache-types-turbo3-turbo4.patch

---
 common/arg.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/arg.cpp b/common/arg.cpp
index e3e7d5868..7f21a4e52 100644
--- a/common/arg.cpp
+++ b/common/arg.cpp
@@ -313,6 +313,8 @@ const std::vector<ggml_type> kv_cache_types = {
     GGML_TYPE_IQ4_NL,
     GGML_TYPE_Q5_0,
     GGML_TYPE_Q5_1,
+    GGML_TYPE_TURBO3_0,
+    GGML_TYPE_TURBO4_0,
 };
 
 static ggml_type kv_cache_type_from_str(const std::string & s) {
